build.gradle

```diff
@@ -37,7 +37,7 @@ subprojects { project ->
       }
     }
     dependencies {
-      classpath 'com.android.tools.build:gradle:2.2.2'
+      classpath 'com.android.tools.build:gradle:2.3.0-rc1'
```

Comment:
Is this required? Non-final releases are timeboxed and I don't want to commit code that uses pre-release versions because that makes the commit unbuildable after a month or two.